src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/SkipTrackMock.kt

```diff
@@ -0,0 +1,14 @@
+package exchange.dydx.abacus.tests.payloads
+
+object SkipTrackMock {
+    val error = """{
+  "code": 5,
+  "message": "chain id not supported",
+  "details": []
```

Comment:
Do we want to surface the error to the user? @mike-dydx

Reply:
FWIW i'm just following the same error pattern we have for other router requests for now.

In the future for error messages we'll be making an abstraction layer to make these more end-user friendly